scripts/crontab/crontab.tpl

```diff
@@ -17,6 +17,7 @@ HOME = /tmp
 
 # Every hour.
 59 * * * * {{ cron }} escalate_questions
+30 * * * * {{ cron }} send_welcome_emails
```

Comment:
I think you re-ordered this file so everything was chronologically ordered. If that's true, then this line should be before the previous line.